setup.py

```diff
@@ -26,24 +26,30 @@
 except ImportError:
     from distutils.core import setup, Extension
 from distutils.core import Command
+from distutils.spawn import find_executable
 from distutils.sysconfig import get_config_var
 from distutils.util import convert_path
 import fnmatch
 import os
+import subprocess
 import sys
+import warnings
 
-from Cython.Distutils import build_ext
-import numpy as np
+try:
+    from Cython.Distutils import build_ext
+except ImportError:
+    raise ImportError('Cython 0.15.1+ is required to install cartopy.')
+try:
+    import numpy as np
+except ImportError:
+    raise ImportError('NumPy 1.6+ is required to install cartopy.')
 
 
-if sys.platform.startswith('win'):
-    def get_config_var(name):
-        return '.'
-    geos = 'geos'
-    extra_extension_args = {}
-else:
-    geos = 'geos_c'
-    extra_extension_args = dict(runtime_library_dirs=[get_config_var('LIBDIR')])
```

Comment:
For future reference, you were correct, and I've moved it to a common location (but since the deletion still happened, GitHub did not hide this comment.)